frontend/src/app/features/in-app-notifications/entry/in-app-notification-entry.component.ts

```diff
@@ -44,34 +40,31 @@ export class InAppNotificationEntryComponent implements OnInit {
 
   workPackage$:Observable<WorkPackageResource>|null = null;
 
+  showDateAlert$:Observable<boolean> = this
+    .storeService
+    .activeReason$
+    .pipe(
+      map((reason) => reason === 'date_alert'),
+      map((dateAlertFiltered) => {
+        const dateAlerts = this.aggregatedNotifications.filter((notification) => notification.reason === 'dateAlert');
```

Comment:
we can save this calculation by returning dateAlertFiltered if available before

Reply:
That's not the same. dateAlertFiltered just returns true if the reason has been filtered (date alert menu is active), but doesn't imply if any notifications are of type date alert